libs/libc/stream/lib_syslogstream.c

```diff
@@ -218,11 +218,6 @@ void lib_syslogstream_close(FAR struct lib_syslogstream_s *stream)
 {
   DEBUGASSERT(stream != NULL);
 
-  if (stream->last_ch != '\n')
```

Comment:
let's restore lib_syslogstream_close to macro like before: https://github.com/apache/nuttx/pull/8004

Reply:
done